node/src/BaseClient.ts

```diff
@@ -4378,6 +4381,70 @@ export class BaseClient {
         return this.createWritePromise(createTouch(keys));
     }
 
+    /**
+     * Pops one or more elements from the first non-empty list from the provided `keys`.
+     *
+     * See https://valkey.io/commands/lmpop/ for more details.
+     *
+     * @remarks When in cluster mode, all `key`s must map to the same hash slot.
+     * @param keys - An array of keys to lists.
+     * @param direction - The direction based on which elements are popped from - see {@link ListDirection}.
+     * @param count - (Optional) The maximum number of popped elements.
+     * @returns A `Map` of key-name mapped array of popped elements.
```

Comment:
```suggestion
     * @returns A `Record` of key-name mapped array of popped elements.
```
Update both + transaction